src/index.html

```diff
@@ -2,50 +2,109 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Product</title>
 
-    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
     <link
       rel="preconnect"
       href="https://fonts.gstatic.com"
       crossorigin="use-credentials"
     >
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap"
+      rel="stylesheet"
+    >
 
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
```

Comment:
I do not think you should change it every time when you have a code line length below 80, not long attribute values and the number of attributes is less than 2. This comment for all such cases.

From Check List:

[CODE STYLE] - Keep your code line length **_below 80_**. It’s not only historical tradition, but also allows your code to fit into one standard screen without horizontal scroll. But do not break the line if it cannot be broken (ex., long links).

[CODE STYLE] - If the HTML-element has long attribute values or number of attributes is **_more than 2_** - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.

Reply:
it's good practice, because if he added here class atribute it's already styled good. So agree with initial code